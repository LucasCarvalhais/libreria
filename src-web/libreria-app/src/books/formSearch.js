import React from 'react';
export const FormSearch = ({ bookId, handleChange, handleSearch }) => <form className="formulario">
    <label>
        <span className="legend">Pesquisar libro: </span>
        <input className="inputForm" type="number" name="bookId" value={bookId} onChange={handleChange} />
    </label>
    <input className="submitButton" type="submit" value="Pesquisar" onClick={handleSearch} />
</form>;
