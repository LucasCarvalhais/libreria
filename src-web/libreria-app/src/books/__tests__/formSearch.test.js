import React from 'react';
import renderer from 'react-test-renderer';
import { FormSearch } from '../formSearch';

test('FormSearch has a valid snapshot', () => {
    const component = renderer.create(
        <FormSearch />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});