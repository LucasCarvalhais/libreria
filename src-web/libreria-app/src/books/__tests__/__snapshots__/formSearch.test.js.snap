// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormSearch has a valid snapshot 1`] = `
<form
  className="formulario"
>
  <label>
    <span
      className="legend"
    >
      Pesquisar libro: 
    </span>
    <input
      className="inputForm"
      name="bookId"
      type="number"
    />
  </label>
  <input
    className="submitButton"
    type="submit"
    value="Pesquisar"
  />
</form>
`;
